src/restaurantSide/pages/auth/ForgotPasswordHost.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import {
+  Grid,
+  Box,
+  TextField,
+  Typography,
+  Button,
+  Snackbar,
+  Alert,
+  IconButton,
+} from '@mui/material';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import { LogoImg } from './styled';
+
+const ForgotPasswordHost = () => {
+  const [email, setEmail] = useState('');
+  const [notification, setNotification] = useState({
+    open: false,
+    severity: '',
+    message: '',
+  });
+
+  const handleForgotPassword = async () => {
+    if (!email) {
+      setNotification({
+        open: true,
+        severity: 'error',
+        message: 'Please enter your email address.',
+      });
+      return;
+    }
+
+    setNotification({
+      open: true,
+      severity: 'success',
+      message: 'Password reset email sent. Check your inbox.',
+    });
+  };
+
+  const handleCloseNotification = () => {
+    setNotification({ ...notification, open: false });
+  };
+
+  const [isFocused] = useState(false);
+
+  return (
+    <Grid container justifyContent='center' alignItems='center' spacing={2} style={{ minHeight: '100vh', maxWidth: '1500px' }}>
```

Comment:
You shouldn't add fixed pixels width to the container, try to figure other ways

Reply:
Did this: <Grid container justifyContent='center' alignItems='center' style={{ minHeight: '100vh' }}>
      {}
      <Grid item xs={12} sm={10} md={8} lg={6} xl={4}>